pysat/_meta.py

```diff
@@ -478,7 +482,12 @@ def __setitem__(self, names, input_data):
 
         elif isinstance(input_data, Meta):
             # dealing with higher order data set
-            
+            # names is only a single name here (by choice for support)
+            if (names in self._ho_data) and (input_data.empty):
```

Comment:
So, if you assign an empty meta object to a meta object label that already has higher order metadata, nothing happens? Is there a case where the user might want to replace an existing ho meta object with a new empty one?